### DISCUSSION TITLE
Most Recent Release But a Few Steps Back

### DISCUSSION THREAD

**Discussion by trubipdontreeopranti** [apprenticeharper/DeDRM_tools#2006] [Q&A]
Hi,

For me, it's always easier to install a version a few steps before the latest version of any kind of software.  I'm not really good with technology! :P :D  

So, I need to know please what would be the release of the DeDRM software that's a bit less than new, but FAIRLY new, and what Calibre version I can use that with?

Thanks!!

**Comment by Dragoon91786** [comment]
I'm still using the Desktop version of **Kindle for PC**—**Version 1.17.0 (44170)**, which can be found online. Sometimes it glitches and I have to attempt to download a book twice, which is why I created some keyboard macros to make downloading the files far easier.

The _current_ **DeDRM_Tools** version found under <a href="https://github.com/apprenticeharper/DeDRM_tools/releases/tag/v7.2.1">[Releases]</a>, **v7.2.1**. This latest version of of **DeDRM Tools** (v7.2.1), _albeit_ **_10-months old_** as of this writing, presently works without issue with the **latest versions** of _**Calibre**_ (**Version: 5.35.0**).

I have not detected _any_ substantive issues with **DeDRM_Tools** in **Calibre** (outside of human-operator-ertor), provided the user has _installed_ the **recommended add-ons/plug-ins** for **Calibre**.
> I have noted some crashing with Calibre, but that appears to be related to the **Calibre Server** feature & to do with plugging my **Android** phone into my computer via USB 3.0/3.1 via USB-C to USB-A, when the phone's developer menu setting have **Debugging Mode** switched ON, as **Calibre** appears unable to properly detect the mobile device (**LG LM-V600AM** running **Android 10.0**). I don't believe this has anything to do with **DeDRM Tools**. Though, it's probably something to keep in mind. :3

I was also able to transfer my **Google Books** library into **Calibre**, and therein get those books transferred into easier to manage formats after downloading the **Adobe** encrypted book files. Twas a royal pain, but I managed.

Would be awesome if we had an **add-on** that could _**automate**_ the process within **Calibre** so we don't have to download the files through one program and then find them on the system, then transfer them, then load them into **Calibre** to be freed of the crippling **DRM**.

If you have any questions, feel free to shoot me a **message here**, as a **DM**, or via <a href="https://discord.com/">**Discord**</a>. (<span style="text-decoration: underline">**Discord**</span>**:** **TheDragonLord#8313**)

You can find specific details by reading: <a href="https://github.com/apprenticeharper/DeDRM_tools/wiki/Exactly-how-to-remove-DRM">_**<span style="text-decoration: underline">[Guide] How to Remove DRM: A Step-By-Step Walkthrough</span>**_</a>

**Comment by ElleKayEm** [comment]
The latest version here is 7.2.1.  There is a version 10.0.2 with many improvements here:  https://github.com/noDRM/DeDRM_tools

Both work with calibre 5.x.
